app/main.py

```diff
@@ -1,214 +1,32 @@
-KNIGHTS = {
```

Comment:
Maybe I miss something, but I can’t find this dict in your updated code. You still need it

Reply:
@AlimFreight  pls move this dict to your code

Reply:
I am really confused.

Reply:
Why I should return this dictionary to the code if I don't use it?

Reply:
To have some example